### DISCUSSION TITLE
Billing_code_type Currently Allowed Codes in Allowed-amount File

### DISCUSSION THREAD

**Discussion by APWU-GRoberts** [CMSgov/price-transparency-guide#103] [Q&A] [answered]
Is the list of "Currently Allowed Codes" and explicit list of allowed codesets?  

We process dental claims that use CDT codes.  Should services and providers submitting claims using CDT codes be included in the file(s)?

**Comment by tdfow** [comment, marked as answer]
This issue talks about adding CDT:  https://github.com/CMSgov/price-transparency-guide/discussions/9
